.github/workflows/ci.yml

```diff
@@ -57,16 +57,17 @@ jobs:
     - name: Get libc version
       run: ldd --versio
 
-    - name: Get branch name for pull request
-      if: ${{ github.event_name == 'pull_request' }}
-      run:  |
-          echo "GIT_BRANCH=$GITHUB_BASE_REF" >> $GITHUB_ENV
-
-    - name: Get branch name for push
+    - name: Get branch name to match with other repos for push
       if: ${{ github.event_name == 'push' }}
       run:  |
-        echo "GIT_BRANCH=${{ github.ref_name }}" >> $GITHUB_ENV
+          echo "GIT_BRANCH=${{ github.ref_name }}" >> $GITHUB_ENV
     
+    - name: Get branch name to match with other repos for pull_request
+      if: ${{ github.event_name == 'pull_request' }}
```

Comment:
This works for the Ubuntu pull request build. It looks like mac needs the same update:

![image](https://github.com/user-attachments/assets/9927a13b-d261-433d-b625-ea795c88f519)

Reply:
Yes updated, sorry for the slow step by step on this PR.